### ISSUE TITLE
ONNX export with EmbeddingBag

### ISSUE THREAD

**Issue by tanguycdls** [pytorch/pytorch#25469]
## 🐛 Bug
I have a dummy model used to test if Embedding Bag is compatible with ONNX or not. The model takes one sparse input for Embedding Bag and one tensor of 2 dim (test both usage).
I tried exporting it in :
- Opset 9 /  operator_export_type=OperatorExportTypes.ONNX -> model loads Ok on ONNX : Onnx runtime:
> GENERAL ERROR : Load model from model.onnx failed:Type Error: Type parameter (T) bound to different types (tensor(float) and tensor(int64) in node (). 

- Opset 10 /  operator_export_type=OperatorExportTypes.ONNX -> ONNX loads it and say OK, ONNX runtime:
> Load model from model.onnx failed:Fatal error: ATen is not a registered function/op
## To Reproduce
```
print(torch.__version__)

class DummyModel(nn.Module):
    def __init__(self):
        super().__init__()
        self.embedd = nn.EmbeddingBag(500, 12)

    def forward(self, x_user):
        user = self.embedd(x_user)
        return torch.cat((user, model), dim=0)
a = DummyModel()
batch_size = 10
from torch.onnx import OperatorExportTypes
onnx_torch.export(a, np.random.randint(0, 7, size=(batch_size, 5)).astype(np.int64),
                  'model.onnx', 
                  input_names='inp',
                  output_names=['output'],
                  dynamic_axes={'inp': {0: 'batch_size'},
                                }, verbose=True,
                  operator_export_type=OperatorExportTypes.ONNX,
                  opset_version=10
                 )
onnx_model = onnx.load("model.onnx")
print(onnx.checker.check_model(onnx_model))
a = onnxruntime.InferenceSession('model.onnx')
```
## Expected behavior
Don't know if EmbeddingBag is usable in ONNX Opset 9 ? If yes use it otherwise warn the user or even better transform it to Embedding + Mean or Sum. 

Thank you !
## Environment

```
Collecting environment information...
PyTorch version: 1.2.0
Is debug build: No
CUDA used to build PyTorch: 10.0.130

OS: Debian GNU/Linux 9 (stretch)
GCC version: Could not collect
CMake version: Could not collect

Python version: 3.6
Is CUDA available: No
CUDA runtime version: No CUDA
GPU models and configuration: No CUDA
Nvidia driver version: No CUDA
cuDNN version: No CUDA

Versions of relevant libraries:
[pip] numpy==1.16.2
[pip] pytorch-ignite==0.2.0
[pip] torch==1.2.0
[conda] _tflow_select             2.3.0                       mkl  
[conda] ignite                    0.2.0                    py36_0    pytorch
[conda] mkl                       2019.4                intel_243    intel
[conda] mkl_fft                   1.0.13           py36h7b7c402_1    intel
[conda] mkl_random                1.0.4            py36h7b7c402_1    intel
[conda] pytorch                   1.2.0           py3.6_cuda10.0.130_cudnn7.6.2_0    pytorch
[conda] tensorflow                1.14.0          mkl_py36h2526735_0  
[conda] tensorflow-base           1.14.0          mkl_py36h7ce6ba3_0
```

**Comment by tanguycdls** [comment]
Hi, I'm still interested by this issue. What would be the best solution to fix it at least temporarly ? 
- When Sparse multlipication are implemented on ONNX we could convert the embedding bag to a sparse multiplication (TF does it for their equivalent to EmbeddingBag) ? https://github.com/onnx/onnx/pull/2019
  - How could we overide the way Torch is converting EmbeddingBag ?
- I also opened the issue https://github.com/pytorch/pytorch/issues/25473 : in caffe2 you can load the model but the dynamic axes look broken ? Would it be easier to fix and use ? 

Thank you for your help,

**Comment by artemru** [comment]
the problem appeared also in https://github.com/facebookresearch/dlrm/issues/12.
